src/terra-dev-site/doc/application/how-to.2/theme-strategy.doc.mdx

```diff
@@ -0,0 +1,166 @@
+import ExampleComponent from './example/ExampleComponent?dev-site-example';
+
+# Theming Guide
+
+Welcome to the Terra theming strategy guide. This guide will walk teams through the recommended approach for theming custom components.
+
+## Summary
+
+Terra supports a default, clinical-lowlight-theme, and orion-fusion-theme. This guide is for theming custom components. Custom components are any non Terra-UI component created by teams that need to be integrated into an application using a supported Terra theme. Teams should not create new themes or theme Terra components.
+
+## Getting Started
+
+To get started, check out the [terra-application](http://engineering.cerner.com/terra-application/components/terra-application/application/reference/components/application-base) documentation. For this guide, it is expected that teams are utilizing an application framework that defines a theme provider. This guide will walk through theming an example component.
+
+## Theme Context
+
+An application's theme is provided through [React context](https://reactjs.org/docs/context.html) by a theme provider. [Application Base](http://engineering.cerner.com/terra-application/components/terra-application/application/reference/components/application-base) initializes a theme provider automatically. The theme value is accessed using the [theme context](http://engineering.cerner.com/terra-application/components/terra-application/application/reference/contexts/theme-context) from terra-application.
+
+```jsx
+import { ThemeContext } from 'terra-application/lib/theme';
+```
+
+Use the theme context to read the active theme value.
+
+```jsx
+import { ThemeContext } from 'terra-application/lib/theme';
+
+const ExampleComponent = () => {
+  // Access the active theme context value.
+  const theme = useContext(ThemeContext);
+};
+```
+
+The active theme name is stored in a `className` key in the context value. To apply the theme, bind the theme className to the component using [classnames](https://github.com/JedWatson/classnames). Some additional documentation for CSS Modules and classNames can be found on the [conventions page](https://engineering.cerner.com/terra-ui/contributing/terra-ui/conventions). Please note that an empty or undefined className indicates the default theme is active or a theme provider is missing. The `className` key maps directly to the anticipated theme name. Using the correct theme className is required.
```

Comment:
should we link directly to classnames/bind? https://github.com/JedWatson/classnames#alternate-bind-version-for-css-modules?

Reply:
Linked to classnames bind: https://github.com/cerner/terra-application/pull/25/commits/2a93997b6888d4c01d6be725178806bdd92ac386